contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1943,20 +1943,86 @@ class tsqlBuilder : public tsqlCommonMutator
 				throw PGErrorWrapperException(ERROR, ERRCODE_INVALID_FUNCTION_DEFINITION, "SELECT statement returning result to a client cannot be used in a function", getLineAndPos(ctx->select_statement_standalone()));
 
 			/* T-SQL doens't allow side-effecting operations in CREATE FUNCTION */
+			/* T-SQL doesn't allow side-effecting operations in CREATE FUNCTION */
```

Comment:
Nit: Two same comments, I guess this was a mere miss.

Reply:
Nope, it's a spelling error correction